.github/workflows/release2.yaml

```diff
@@ -1,32 +1,59 @@
-name: release
+name: Build release
+
+# This is triggered by merging a PR from a branch named release-*. This workflow builds and publishes the release. Then it changes
+# the version to *-SNAPSHOT in various Java-related files.
 
 on:
-  push:
-    tags:
-    - '*'
+  pull_request:
+    types:
+    - closed
 
 jobs:
-  release:
+  github-upload:
+    if: github.event.pull_request.merged
     runs-on: ubuntu-18.04
     needs:
       - build
       - android
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/create-release@v1
-      id: release
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
-        tag_name: ${{ github.ref }}
-        release_name: Version ${{ github.ref }}
+        # base_ref will typically be master. We'll need to commit a change to that branch and push it.
+        ref: ${{ github.base_ref }}
+        # If we use the default GITHUB_TOKEN then CI won't run after we push our changes. This DEPLOYMENT_TOKEN is a personal access
+        # token with "repo" permissions.
+        token: ${{ secrets.DEPLOYMENT_TOKEN }}
     - name: Decrypt PGP key
       uses: IronCoreLabs/ironhide-actions/decrypt@v1
       with:
         keys: ${{ secrets.IRONHIDE_KEYS }}
         input: .github/signing-key.asc.iron
     - name: Import PGP key
       run: gpg --batch --import .github/signing-key.asc
+    - name: Configure git
+      run: |
+        git config --local user.email bot@cjones.org
```

Comment:
Is this the user we want here?

Reply:
Whoops! Thanks for catching that. I did a bunch of testing in a repo on my personal account. Will fix.